src/block-components/container-div/index.js

```diff
@@ -22,11 +23,12 @@ export const ContainerDiv = props => {
 	const instanceId = useQueryLoopInstanceId( attributes.uniqueId )
 	let uniqueBlockClass = getUniqueBlockClass( attributes.uniqueId )
 	uniqueBlockClass = instanceId ? uniqueBlockClass + `-${ instanceId }` : uniqueBlockClass
+	const uniqueContainerClass = applyFilters( 'stackable.container-div.uniqueClass.edit', `${ uniqueBlockClass }-container`, uniqueBlockClass )
```

Comment:
Same here, you most probably forgot to also pass the attributes as a filter argument.